demisto_sdk/commands/common/tools.py

```diff
@@ -653,6 +655,39 @@ def pack_name_to_path(pack_name):
     return os.path.join(PACKS_DIR, pack_name)
 
 
+def get_pack_ignore_file_path(pack_name):
+    return os.path.join(PACKS_DIR, pack_name, PACKS_PACK_IGNORE_FILE_NAME)
+
+
+def get_ignore_pack_skipped_tests(pack_name: str) -> set:
+    ignored_tests_set = set()
+    if pack_name:
+        pack_ignore_path = get_pack_ignore_file_path(pack_name)
+
+        if os.path.isfile(pack_ignore_path):
+            try:
+                config = ConfigParser(allow_no_value=True)
+                config.read(pack_ignore_path)
+
+                # create file specific ignored tests set
+                for section in config.sections():
+                    if section.startswith("file:"):
+                        file_name = section[5:]
+                        for key in config[section]:
+                            if key == 'ignore':
+                                ignore_list = str(config[section][key]).split(',')
+                                if PACK_IGNORE_TEST_FLAG in ignore_list:
+                                    path = os.path.join(PACKS_DIR, pack_name, TEST_PLAYBOOKS_DIR, file_name)
```

Comment:
can you please add inline comments(preferable with an examples)
or even docstring with an example to the func
as it is hard to follow.